### PR TITLE
[v4.4.1-crio] Bump c/common to v0.52.0

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
As the title says, bumping the c/common package to v0.52.0

in the v4.4.1-crio branch via a request from the CRI-O team.

Addresses: https://issues.redhat.com/browse/OCPBUGS-32144

[NO NEW TESTS NEEDED]

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@kwilczynski PTAL

  This is dragging a ton of stuff in, I hope it's not too much.

File: go.mod [hunk 1]
```diff
@@ -8,21 +8,21 @@ require (
 	github.com/buger/goterm v1.0.4
 	github.com/checkpoint-restore/checkpointctl v0.0.0-20220321135231-33f4a66335f0
 	github.com/checkpoint-restore/go-criu/v6 v6.3.0
-	github.com/container-orchestrated-devices/container-device-interface v0.5.3
+	github.com/container-orchestrated-devices/container-device-interface v0.5.4
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.2.0
 	github.com/containers/buildah v1.29.3
-	github.com/containers/common v0.51.3
+	github.com/containers/common v0.52.0
```
Comment (kwilczynski) [inline go.mod:15]: This aligns with CRI-O release `1.25`. Newer releases are already on version `0.53.0` or newer.

**Comment by kwilczynski** [comment]
> This is dragging a ton of stuff in, I hope it's not too much.

@TomSweeneyRedHat, the two most important dependencies here for us are the `c/common` and `c/image`, and updates to both look OK from our point of view.

We are good to go, I believe.

**Comment by kwilczynski** [comment]
@TomSweeneyRedHat, even though we look good from our key dependencies point of view, we have some anciliary dependency issues per:

```go
vendor/github.com/openshift/imagebuilder/dockerfile/parser/parser.go:122:12: undefined: system.LCOWSupported
vendor/github.com/openshift/imagebuilder/dockerfile/parser/parser.go:157:12: undefined: system.LCOWSupported
```

Let me have a look as to how we resolved this in CRI-O. I believe a dependency update would fix this.

**Comment by kwilczynski** [comment]
> @TomSweeneyRedHat, even though we look good from our key dependencies point of view, we have some anciliary dependency issues per:
> 
> ```go
> vendor/github.com/openshift/imagebuilder/dockerfile/parser/parser.go:122:12: undefined: system.LCOWSupported
> vendor/github.com/openshift/imagebuilder/dockerfile/parser/parser.go:157:12: undefined: system.LCOWSupported
> ```
> 
> Let me have a look as to how we resolved this in CRI-O. I believe a dependency update would fix this.

This missing method issue has been fixed in the following release of the `openshift/imagebuilder` package:

- https://github.com/openshift/imagebuilder/issues/238

We should upgrade the dependency to version `1.2.4` (a proper full release) or newer. We also use version `1.2.4` of the said package in CRI-O `1.25`, where this failure does not occur.

File: go.mod [hunk 2]
```diff
@@ -38,28 +38,28 @@ require (
...
+	github.com/onsi/gomega v1.27.6
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2
+	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
 	github.com/opencontainers/runc v1.1.12
 	github.com/opencontainers/runtime-spec v1.1.0-rc.1
-	github.com/opencontainers/runtime-tools v0.9.1-0.20221014010322-58c91d646d86
+	github.com/opencontainers/runtime-tools v0.9.1-0.20221107090550-2e043c6bd626
 	github.com/opencontainers/selinux v1.11.0
 	github.com/openshift/imagebuilder v1.2.4-0.20220711175835-4151e43600df
```
Comment (kwilczynski) [inline go.mod:51]: @TomSweeneyRedHat, if possible, this needs a version bump to:

```suggestion
	github.com/openshift/imagebuilder v1.2.4
```

This looks confusing; however, it's how this project tagged their release.

**Comment by TomSweeneyRedHat** [comment]
Replaced by: https://github.com/containers/podman/pull/22944

**Pull request closed without merging**